src/ServiceControl.Audit.Persistence.RavenDB/RavenAttachmentsBodyStorage.cs

```diff
@@ -11,15 +12,15 @@ class RavenAttachmentsBodyStorage(
         int settingsMaxBodySizeToStore)
         : IBodyStorage
     {
-        public Task Store(string bodyId, string contentType, int bodySize, Stream bodyStream)
+        public Task Store(string bodyId, string contentType, int bodySize, Stream bodyStream, CancellationToken cancellationToken)
         {
             if (bodySize > settingsMaxBodySizeToStore)
             {
                 return Task.CompletedTask;
             }
 
             return bulkInsert.AttachmentsFor(bodyId)
-                .StoreAsync("body", bodyStream, contentType);
+                .StoreAsync("body", bodyStream, contentType, cancellationToken);
         }
 
         public async Task<StreamResult> TryFetch(string bodyId)
```

Comment:
Should this have a token as well? (I know this is on a different code path)

Reply:
@andreasohlund there are many more places that can support cancellation but this PR tries to focus on the majority of the audit/error ingestion IO.

Reply:
Added additional comments to ensure all audit storage operations support cancellation